tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
```

Comment:
Please factor these out into a separate pull request. It's entirely unrelated to adding AppVeyor CI.